### PR TITLE
Feature/39906

### PR DESCRIPTION
**Pull request by Aekal**
dev-39906

### CODE REVIEW

File: components/01-globals/grid/_grid.scss [hunk 1]
```diff
@@ -7,6 +7,10 @@ $grid__side-gutter: false !default;
     justify-content: space-between;
     align-items: stretch;
 
+    &--align-start {
+        justify-content: flex-start;
+    }
+
```
Comment (szafran89) [inline components/01-globals/grid/_grid.scss]: IMO we should remove this modifier

File: components/03-modules/sidebar-block/_sidebar-block.scss [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+$sidebar-block__item-margin-bottom         : $spacer !default;
+$sidebar-block__index-margin               : 0 $spacer !default;
+$sidebar-block__index-font-weight          : $font-weight-bold !default;
+$sidebar-block__index-min-width            : $spacer--medium !default;
+$sidebar-block__remove-padding             : $spacer--medium !default;
+$sidebar-block__remove-margin-right        : $spacer !default;
+$sidebar-block__remove-icon-margin         : 0 auto !default;
+$sidebar-block__remove-icon-padding        : 0 !default;
+$sidebar-block__icon-fill                  : $red !default;
+$sidebar-block__icon-size                  : $spacer--medium !default;
```
Comment (szafran89) [inline components/03-modules/sidebar-block/_sidebar-block.scss]: icon size should not get value from spacer. Please use a value in px's.

File: components/03-modules/sidebar-block/_sidebar-block.scss [hunk 3]
```diff
@@ -0,0 +1,111 @@
...
+
+    &__checkbox {
+        margin: $sidebar-block__checkbox-margin;
+    }
+
+    &__hidden {
+        display: none;
+    }
+
+    .price-as-configured {
```
Comment (szafran89) [inline components/03-modules/sidebar-block/_sidebar-block.scss]: Can we change to BEM?

File: components/03-modules/sidebar-block/_sidebar-block.scss [hunk 4]
```diff
@@ -0,0 +1,111 @@
...
+        margin: $sidebar-block__price-margin;
+        font-weight: $sidebar-block__price-font-weight;
+    }
+
+    .message {
```
Comment (szafran89) [inline components/03-modules/sidebar-block/_sidebar-block.scss]: Can we change to BEM?

File: components/04-views/dashboard/wishlist/_wishlist.scss [hunk 5]
```diff
@@ -0,0 +1,120 @@
...
+
+    &__remove {
+        margin-left: $wishlist__remove-margin-left;
+    }
+
+    &__comment {
+        margin: $wishlist__comment-margin;
+    }
+
+    .action {
```
Comment (szafran89) [inline components/04-views/dashboard/wishlist/_wishlist.scss]: Can we change to BEM class?

File: components/01-globals/grid/_grid.scss (hunk 1 above)
Comment (Aekal) [reply components/01-globals/grid/_grid.scss]: Moved to wishlist component

File: components/03-modules/sidebar-block/_sidebar-block.scss (hunk 2 above)
Comment (Aekal) [reply components/03-modules/sidebar-block/_sidebar-block.scss]: Done

File: components/03-modules/sidebar-block/_sidebar-block.scss (hunk 4 above)
Comment (Aekal) [reply components/03-modules/sidebar-block/_sidebar-block.scss]: Done

File: components/04-views/dashboard/wishlist/_wishlist.scss (hunk 5 above)
Comment (Aekal) [reply components/04-views/dashboard/wishlist/_wishlist.scss]: Done

**Pull request merged**